website/static/js/passwordForms.js

```diff
@@ -119,6 +119,30 @@ var ChangePasswordViewModel = oop.extend(BaseViewModel, {
 var SetPasswordViewModel = oop.extend(BaseViewModel, {
     constructor: function () {
         var self = this;
+        // Call constructor at the beginning so that self.password exists
+        self.super.constructor.call(this);
```

Comment:
Oh, but this class doesn't override `getValidatedFields`, so I think this is OK. 👍